documentation/website/docs/installation.md

```diff
@@ -54,6 +54,21 @@ This will generate a `Makefile` in your checkout directory. You can subsequently
 $ make
 $ make test
 ```
+### Testing changes to the Python Client
+In a virtualenv, install dependencies with `requirements.txt` and run python tests to make sure everything is set up correctly
+
+```bash
+$ cd /path/to/pyre-check
+$ pip install -r requirements.txt
+$ ./scripts/run-python-tests.sh
+```
+When installing and running `pyre` from PyPi, the entry point to the executable is actually `client/pyre.py`. To be able to run this file from anywhere, add the `pyre-check` directory to `PYTHONPATH` and subsequently assign `client.pyre` an alias to invoke the binary from anywhere
+
+```bash
+$ echo "alias pyre='PYTHONPATH=\"/path/to/pyre-check:\$PYTHONPATH\" python -m client.pyre'" >> ~/.bashrc
```

Comment:
Just checking, you tested this latest variation on your own machine and it worked, right?

Reply:
I believe so, yes. `pyre` ran fine after that command. Moreover, my `.zshrc` file gets updated with the following line
![image](https://user-images.githubusercontent.com/17424778/109998429-38875a80-7d37-11eb-8056-b80e240c28b5.png)
after running the command, which i think is correct